client/client.go

```diff
@@ -211,6 +211,20 @@ func (e *Error) Error() string {
 	return e.Message
 }
 
+const (
+	ErrorKindTwoFactorRequired = "two-factor-required"
+	ErrorKindTwoFactorFailed   = "two-factor-failed"
+)
+
+func IsTwoFactorError(err error) bool {
```

Comment:
Not a big deal but a ```// comment``` would be nice, perhaps:

// IsTwoFactorError checks if the error is related to two factor authentication.